src/groovy/heim/tasks/DataFetchTask.groovy

```diff
@@ -246,7 +246,7 @@ class DataFetchTask extends AbstractTask {
                 "if (!exists('loaded_variables')) { loaded_variables <- list() }",
                 """
                 loaded_variables[['$escapedLabel']] <- read.csv(
-                               '$escapedFilename', sep = "\t", header = TRUE);
+                               '$escapedFilename', sep = "\t", header = TRUE, stringsAsFactors = FALSE);
                 names(loaded_variables)""",
```

Comment:
a tab-separated .csv (comma separated values) file?
I think read.csv is for csv files, read.table is the more general variant. However, if this works...